build.zig

```diff
@@ -24,16 +24,18 @@ pub fn build(b: *std.build.Builder) !void {
         BuildConfig{ .name = "boards.stm32f3discovery", .backing = Backing{ .board = boards.stm32f3discovery } },
         BuildConfig{ .name = "boards.stm32f4discovery", .backing = Backing{ .board = boards.stm32f4discovery } },
         BuildConfig{ .name = "boards.stm32f429idiscovery", .backing = Backing{ .board = boards.stm32f429idiscovery }, .supports_uart_test = false },
+        BuildConfig{ .name = "chips.gd32vf103x8", .backing = Backing{ .chip = chips.gd32vf103x8 }, .supports_interrupt_test = false },
+        BuildConfig{ .name = "boards.longan_nano", .backing = Backing{ .board = boards.longan_nano }, .supports_interrupt_test = false },
     };
 
-    const Test = struct { name: []const u8, source: []const u8, uses_uart: bool = false, on_avr: bool = true };
+    const Test = struct { name: []const u8, source: []const u8, uses_uart: bool = false, on_riscv32: bool = true, on_avr: bool = true };
```

Comment:
why do we have both a `supports_interrupt_test` and `on_riscv32`? isn't one enough? otherwise it will get confusing have to turn off both when we get this working.